src/hotspot/share/jvmci/jvmci_globals.cpp

```diff
@@ -79,7 +79,7 @@ bool JVMCIGlobals::check_jvmci_flags_are_consistent() {
   CHECK_NOT_SET(JVMCIHostThreads,             UseJVMCICompiler)
   CHECK_NOT_SET(LibJVMCICompilerThreadHidden, UseJVMCICompiler)
 
-  if (UseJVMCICompiler) {
+  if (EnableJVMCI) {
```

Comment:
This needs to be `EnableJVMCI || UseJVMCICompiler` (since deriving `EnableJVMCI` from `UseJVMCICompiler` is only done [below](https://github.com/openjdk/jdk/blob/ab06a878f888827026424530781f0af414a8a611/src/hotspot/share/jvmci/jvmci_globals.cpp#L96).

Reply:
I see, `FLAG_SET_DEFAULT(EnableJVMCI, true)` on [line 99](https://github.com/openjdk/jdk/blob/78f576192e815f957db93f5f8cb3763a35474381/src/hotspot/share/jvmci/jvmci_globals.cpp#L99)

Reply:
It's not obvious to me how that's clearer than just expanding the guard on line 82 to be `EnableJVMCI || UseJVMCICompiler`.

Reply:
Now that I see your change and understand what you meant, it is better - thanks.